build/Packages.props

```diff
@@ -2,57 +2,57 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <MSBuildPackageVersion>15.6.82</MSBuildPackageVersion>
-    <NuGetPackageVersion>4.3.0</NuGetPackageVersion>
-    <RoslynPackageVersion>2.8.0</RoslynPackageVersion>
+    <MSBuildPackageVersion>15.7.179</MSBuildPackageVersion>
+    <NuGetPackageVersion>4.7.0</NuGetPackageVersion>
+    <RoslynPackageVersion>2.8.2</RoslynPackageVersion>
     <XunitPackageVersion>2.3.1</XunitPackageVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Update="Cake.Scripting.Transport" Version="0.2.0" />
-    
+    <PackageReference Update="Cake.Scripting.Transport" Version="0.3.0" />
+
     <PackageReference Update="Dotnet.Script.DependencyModel" Version="0.6.1" />
     <PackageReference Update="Dotnet.Script.DependencyModel.NuGet" Version="0.6.2" />
 
-    <PackageReference Update="Microsoft.AspNetCore.Diagnostics" Version="1.1.0" />
-    <PackageReference Update="Microsoft.AspNetCore.Hosting" Version="1.1.0" />
-    <PackageReference Update="Microsoft.AspNetCore.Http.Features" Version="1.1.0" />
-    <PackageReference Update="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.0" />
+    <PackageReference Update="McMaster.Extensions.CommandLineUtils" Version="2.2.4" />
```

Comment:
nice. got caught by surprise with this one the other day when upgrading another project - could not find the new version, only to discover it is not supported by MS anymore 😀